src/routes/admin_routes.js

```diff
@@ -202,11 +202,11 @@ router.get('/featuredcubes', ensureAdmin, async (req, res) => {
 
   return render(req, res, 'FeaturedCubesQueuePage', {
     cubes: sortedCubes,
-    lastRotation: featured[0].featuredOn,
+    lastRotation: featured.length > 0 ? featured[0].featuredOn : new Date(0).valueOf(),
```

Comment:
Edge case that probably would only affect local development. If nothing in queue then the last rotation is 1969